duo-client/src/main/java/com/duosecurity/client/Http.java

```diff
@@ -313,6 +322,11 @@ private String canonQueryString()
     return Util.join(args.toArray(), "&");
   }
 
+  private String canonJson(){
+    JSONObject jsonBody = new JSONObject(params);
+    return  jsonBody.toString();
```

Comment:
Looks like `toString()` method does not allow add `separators` like python does. So we need custom json to string method in util now